solutions/swb-app/src/generateRouter.ts

```diff
@@ -84,6 +88,17 @@ export function generateRouter(apiRouteConfig: ApiRouteConfig): Express {
   app.use(verifyToken(authenticationService, { ignoredRoutes: staticRoutesIgnored, loggingService: logger }));
   app.use(withAuth(authorizationService, { logger: logger }));
 
+  // Auditing
+  const continueOnError = false;
+  const requiredAuditValues = ['actor', 'source'];
+  // Masking fields 'codeVerifier' and 'code' because they're in the body of '/token' API
+  const fieldsToMask = ['user', 'password', 'accessKey', 'code', 'codeVerifier'];
```

Comment:
Is it possible to move these into configuration? E.g. JSON file. This way it would be easier to modify if we identify that we don't want to log personal information or secrets

Reply:
yep, I'll do that